File: Java/UberBlack.java
package Java;

class UberBlack extends Car {
    // We declare the attributes that are only UberBlack's attributes
    Map<String, Map<String, Integer>> typeCarAccepted;



    //todo Constructor method:
    public UberBlack(String license, Account driver){
        super(license, driver); // As if you called the constructor method of the car class and printDataCar method.
    }
}

File: PHP/uberBlack.php
<?php
require_once("car.php"); 
class UberBlack extends Car { //Declare two global attributes which were pendings to UberBlack model
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, Account $driver, $typeCarAccepted, $seatsMaterial){
        parent::__construct($license, $driver);  
        $this->typeCarAccepted = $typeCarAccepted; 
        $this->seatsMaterial = $seatsMaterial; 
    }
}
?>